src/Domain/Model/DynamoDbModel.php

```diff
@@ -183,6 +183,10 @@ public function save(array $options = [])
             $this->fireModelEvent('creating');
         }
 
+        if ($this->usesTimestamps()) {
```

Comment:
Would be better to move this after the `$this->client->putItem();` function call, so that the timestamps gets updated when the operation is successful.